package.json

```diff
@@ -30,10 +30,13 @@
     "@tailwindcss/postcss7-compat": "^2.2.17",
     "@types/greensock": "^1.15.32",
     "core-js": "^3.25.0",
+    "expr-eval": "^2.0.2",
     "gsap": "^3.11.1",
     "harp": "^0.41.0",
+    "i": "^0.3.7",
     "lodash.get": "^4.4.2",
     "lodash.merge": "^4.6.2",
+    "npm": "^8.19.2",
```

Comment:
Why did we update npm? Will doing this clash with OB that has strict engines set therefore making development across both projects harder?

Reply:
I probably typed `npm i npm i` or something funky.